libcontainer/factory_nabla.go

```diff
@@ -86,6 +87,17 @@ func applyPauseHack(config *configs.Config, containerRoot string) (*configs.Conf
 func createRootfsISO(config *configs.Config, containerRoot string) (string, error) {
 	rootfsPath := config.Rootfs
 	targetISOPath := filepath.Join(containerRoot, "rootfs.iso")
+	for _, mount := range config.Mounts {
+		if (mount.Destination == "/etc/resolv.conf") ||
+			(mount.Destination == "/etc/hosts") ||
+			(mount.Destination == "/etc/hostname") {
+			cmd := exec.Command("cp", mount.Destination, rootfsPath + "/etc/")
+			var err = cmd.Run()
+			if err != nil {
+				return "", errors.Wrap(err, "Unable to copy resolv.conf")
```

Comment:
Error message should be more generic to encompass other failure cases for other `/etc/` files.